test/test.js

```diff
@@ -460,6 +461,45 @@ describe('expressWinston', function () {
             result.transportInvoked.should.eql(false);
           });
         });
+
+        describe('when invoked on a route that should be ignored (options.ignoreRoute)', function () {
+          var result;
+
+          before(function (done) {
+            setUp({
+              url: '/is-not-logged'
+            });
+            req.skip = true;
+            var test = {
+              req: req,
+              res: res,
+              log: {}
+            };
+
+            function next(_req, _res, next) {
+              res.end('{ "message": "Hi!  I\'m a chunk!" }');
+              result = test;
+              return done();
+            };
+
+            var middleware = expressWinston.logger({
+              transports: [new MockTransport(test)],
+              ignoreRoute: function (req, res) {
+                return req.skip === true && req.url.match(/^\/is-not-log/);
```

Comment:
It was to illustrate that any property added to request (by a middleware for example) can be used in ignoreRoute... I guess it is not clear. I can just keep the req.url check.